backend/Bindings/Edna.Bindings.LtiAdvantage/Services/AccessTokenService.cs

```diff
@@ -38,7 +38,7 @@ public async Task<TokenResponse> GetAccessTokenAsync(string clientId, string acc
             var payload = new JwtPayload();
             payload.AddClaim(new Claim(JwtRegisteredClaimNames.Iss, clientId));
             payload.AddClaim(new Claim(JwtRegisteredClaimNames.Sub, clientId));
-            payload.AddClaim(new Claim(JwtRegisteredClaimNames.Aud, accessTokenEndpoint));

```

Comment:
Is it possible to perform this ternary check in the platform itself? Passing it all over across all these functions could be avoided if we could update the platform to store correct Aud itself.

Reply:
hi @adbindal, I would have been happy to, if you could explain how I could do it. the platform is not being passed to the getaccesstokenasync. 
So the way I saw it, even if I moved the check to the platform, I would still have to pass the audience all over the place, because how else do you get the audience to the method? eg, look at the method declaration, it needs client id, accesstokenendpoint being passed all over the place..

Reply:
Sure. I've kind of lost track of the code in my mind so unable to talk too much in terms of functions and files to change. The aim was to modify the Platform Model to store the Audience correctly instead of storing both AccessTokenUrl & Audience. This way we would only be passing `audience` param to the `AccessTokenService`. I'd go ahead and re-read the code to remember those linkages in the code, in order to help you better.

Reply:
I think the issue here is you actually need both parameters 
accesstokenendpoint is necessary as the target address of where to request token from, like on line 51 of that file. 
And audience is necessary for jwtRegisteredClaimNames.Aud. 

I would almost consider it a bug, that the accessTokenService was using the accesstokenendpoint for the JwtRegisteredClaimsName.Aud. in my original post, I mentioned essentially that
The imsglobal spec basically mentioned that it's common to use the tokenendpoint as the Audience, but that it's not a given and it may use any other value. in the implementation here previously, it was basically hard coded to have thte audience as the tokenEndpoint. so while some LMS' like blackboard or Moodle, will send the endpoint as the audience by default, others like Brightspace cannot use the same value.

Reply:
The current changes looks fine for now

Reply:
Later on, we can start passing the platform object